packages/icons/src/order-last.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const OrderLastIcon = React.forwardRef<SVGSVGElement, IconProps>(
+  ({ color = "currentColor", ...props }, forwardedRef) => {
+    return (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 16 16"
+        width="16"
+        height="16"
+        {...props}
+        ref={forwardedRef}
+        style={{ transform: "scale(-1)" }}
+      >
+        <mask id="a" fill="#fff">
+          <rect width="4" height="8" x="10.9" y="4" rx="1" />
+        </mask>
+        <rect
+          width="4"
+          height="8"
+          x="10.9"
+          y="4"
+          stroke={color}
+          strokeWidth="2.2"
+          mask="url(#a)"
+          rx="1"
+        />
+        <mask id="b" fill="#fff">
```

Comment:
this one seems like it's doing something, so we can't just remove it, we need the plugin